modulesync.gemspec

```
@@ -26,6 +26,6 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'git', '~>1.3'
   spec.add_runtime_dependency 'gitlab', '~>4.0'
   spec.add_runtime_dependency 'octokit', '~>4.0'
-  spec.add_runtime_dependency 'puppet-blacksmith', '~>3.0'
+  spec.add_runtime_dependency 'puppet-blacksmith', '>= 3.0'
```

Should we at least set an upper bound?
```suggestion
  spec.add_runtime_dependency 'puppet-blacksmith', '>= 3.0', '< 7'
```